templates/course/lesson-count.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Lessons count in a Course.
+ *
+ * @package LifterLMS/Templates
+ *
+ * @since [version]
+ * @version [version]
+ *
+ * @author      LifterLMS
+ * @package     LifterLMS/Templates
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+global $post;
+
+$course = new LLMS_Course( $post );
+$lessons_count = $course->get_lessons_count();
+?>
+
+<div class="llms-meta llms-lessons-count">
+	<p><?php printf( __( 'Number of Lessons: <span class="lessons-count">%u</span>', 'lifterlms' ), $lessons_count ); ?></p>
+</div>
```

Comment:
Our templates, and PHP files in general, end with a blank line.
Is there a reason why this doesn't have it?

Reply:
the file does have a newline character at the end. It's just that GitHub is not showing that.
And the CS checker is not showing any errors (example - File must end with a newline character (Generic.Files.EndFileNewline.NotFound) ) on this file.